server/service/IndividualServicesService.js

```diff
@@ -140,13 +140,15 @@ exports.bequeathYourDataAndDie = function (body, user, originator, xCorrelator,
       let applicationPort = body["new-application-port"];
       
       
-    let newHttpClientLtpUuid = await resolveHttpClient(FcportValue)
+    let httpClientUuidList = await resolveHttpClient(FcportValue)
```

Comment:
As , it is not list and we are also fetching other uuids not only for  http client can we change the naming convention more precisely for httpclientuuidlist as well as method name

Reply:
Done